homeassistant/components/openweathermap/sensor.py

```diff
@@ -98,10 +95,5 @@ def state(self):
         """Return the state of the device."""
         forecasts = self._weather_coordinator.data.get(ATTR_API_FORECAST)
         if forecasts is not None and len(forecasts) > 0:
-            value = forecasts[0].get(self._sensor_type, None)
-            if self._device_class is DEVICE_CLASS_TIMESTAMP:
-                value = datetime.datetime.fromtimestamp(
-                    value, datetime.timezone.utc
-                ).isoformat()
-            return value
+            return forecasts[0].get(self._sensor_type, None)
```

Comment:
Reversal to coding from before 2021.1.2 since now handled by the weather updater.